release-notes/major12/12.3.0/valtimo-backend-libraries.md

```diff
@@ -46,15 +46,39 @@ Instructions on how to migrate to this version of Valtimo can be found [here](mi
 
 The following was deprecated:
 
-* **Deprecation1**
-
-  X was deprecated and is replaced with Y.
-
-* **Deprecation2**
-
-  X was deprecated and is replaced with Y.
-
-Instructions on how to migrate to this version of Valtimo can be found [here](migration.md).
+* **WebClient is no longer the default HTTP client**
+    
+    Transitioned from using the WebClient class to RestClient. For more details, see the [overview](https://docs.spring.io/spring-framework/reference/integration/rest-clients.html).
+    
+    The primary reason for this switch is to adopt a simpler client that does not rely on reactive programming concepts. Most calls in Valtimo require synchronous execution, making WebClient less suitable.
+    
+    Constructors in affected classes now use RestClient.Builder instead of WebClient.Builder.
+    
+    The default HTTP client is now Apache Client v5.x.
+    For more details on how to override/adjust this, see [ApacheRequestFactoryCustomizer](https://github.com/valtimo-platform/valtimo-backend-libraries/blob/b0941b2ca161601094203b38ac639f1a809988a8/contract/src/main/kotlin/com/ritense/valtimo/contract/client/ApacheRequestFactoryCustomizer.kt).
+    See also [Spring docs](https://docs.spring.io/spring-boot/reference/io/rest-client.html#io.rest-client.restclient.customization)
+    
+    A new configuration class has been introduced to modify two main timeout settings.
```

Comment:
Since there's a new way of doing things, we should also provide proper documentation for that. I'm thinking this should go in the reference section, explaining what properties can be configured to set up your rest client. What do you think?